src/scss/_brand.scss

```diff
@@ -0,0 +1,58 @@
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oSteppedProgressGet($variables, $from: null) {
+	@return oBrandGet($component: 'o-stepped-progress', $variables: $variables, $from: $from);
+}
+
+/// Helper for `o-brand` function.
+/// @access private
+@function _oSteppedProgressSupports($variant) {
+	@return oBrandSupportsVariant($component: 'o-stepped-progress', $variant: $variant);
+}
+
+// Define shared brand configurations
+$_o-stepped-progress-shared-brand-config: (
+	font-scale: -1,
+	step-size: 22px,
+	step-border-size: 2px,
+	label-max-width: 140px,
```

Comment:
An `em` unit might be a little better here, to accommodate a [custom typography scale](https://github.com/Financial-Times/o-typography/pull/166).